tests/config/serve.json

```diff
@@ -0,0 +1,7 @@
+{
+  "public": "tests/fixtures",
+  "cleanUrls": false,
+  "redirects": [
```

Comment:
do we need the redirects directive here?

Reply:
We currently don't test the expected behavior for redirected requests, but I think we should. For example, 302 redirects probably shouldn't preload, but we don't know the exact status code, so we should discard all redirected responses anyway.